apps/prairielearn/package.json

```
@@ -73,7 +73,7 @@
     "blocked": "^1.3.0",
     "blocked-at": "^1.2.0",
     "body-parser": "^1.20.3",
-    "bootstrap": "5.2.3",
+    "bootstrap": "5.3.5",
```

Nit: presumably it doesn't need to be pinned anymore?

```suggestion
    "bootstrap": "^5.3.5",
```

---

Good call, done in 320bbc1.